README.md

```diff
@@ -1,2 +1,16 @@
+
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+```
+for i (initialize i to 0, run this for loop 'WordLength' times, increment 'i' each run)
+	for j (init j to 0, run this for loop 'WordLength' times, increment 'j' each run)
+		if letter i of GuessedWord matches letter j of SecretWord
+			check the matching letter is in the same place by checking the integer i and j are the same value
+				if it is, increment the Score
```

Comment:
This would work, however it is inefficient -- you have nested loops giving O(n^2) runtime, whereas it is possible to solve this problem in O(n) time:
```
for i = 0, ..., wordLength-1
   if letter i of guessedWord matches letter i of secretWord
      increment score
```